consensus/consensus_service.go

```diff
@@ -348,7 +348,9 @@ func (consensus *Consensus) UpdateConsensusInformation() Mode {
 	}
 
 	if consensus.ChainReader.Config().IsFiveSeconds(curEpoch) {
-		consensus.BlockPeriod = time.Duration(5 * time.Second)
+		consensus.BlockPeriod = 5 * time.Second
+	} else {
+		consensus.BlockPeriod = 8 * time.Second
```

Comment:
no need to add else. In fact, we can remove this whole if condition, as we will not go back to 8 seconds anymore across all networks.

Reply:
Removed logic. Thanks for advice.